src/test/java/org/jboss/arquillian/extension/jacoco/test/unit/FilterComposerTestCase.java

```diff
@@ -68,36 +68,36 @@ public void should_exclude_asset_not_matching_include_pattern() throws Exception
    }
 
    @Test
-   public void should_include_asset_not_matching_exclude_pattern() throws Exception
+   public void should_include_asset_matching_one_of_include_patterns() throws Exception
    {
       // given
-      final Filter<ArchivePath> filter = createComposedFilter(null, "org.arquillian.extension.jacoco.client.*");
+      final Filter<ArchivePath> filter = createComposedFilter("org.arquillian.extension.jacoco.client.*, org.arquillian.extension.integration.*", null);
 
       // when
-      final boolean include = filter.include(ArchivePaths.create("org/arquillian/extension/jacoco/container/CoverageDataCommand.class"));
+      final boolean include = filter.include(ArchivePaths.create("org/arquillian/extension/jacoco/client/ManifestAsset.class"));
```

Comment:
https://github.com/arquillian/arquillian-extension-jacoco/blob/760571097351832b8a5a1d3cd9d36ea6a3fc9de7/src/main/java/org/jboss/arquillian/extension/jacoco/client/instrumentation/ManifestAsset.java is it?